Makefile

```diff
@@ -361,7 +361,7 @@ test-ci-js: | clear-stalled
 test-ci: LOGLEVEL := silent
 test-ci: | clear-stalled build-addons build-addons-napi
 	out/Release/cctest --gtest_output=tap:cctest.tap
-	$(PYTHON) tools/test.py $(PARALLEL_ARGS) -p tap --logfile test.tap \
+	$(PYTHON) tools/test.py $(PARALLEL_ARGS) -p oneline --logfile test.tap \
```

Comment:
nitpick: should we rename the output file too?

Reply:
Right, updated – there is no logfile for non-tap output anyway :)